### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by AymanAlqoqa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,10 +31,27 @@ South Londoners(Member):
 - Post an opportunity [#2](https://github.com/FACG6/city-in-the-south/issues/2).
 - Update profile [#3](https://github.com/FACG6/city-in-the-south/issues/3).
 
-## TWEET
+## Prototype:
+
+Website prototype is [here](https://www.figma.com/file/Pom80HoRieQ24tSmTzayO4WP/City-of-south?node-id=122%3A474).
```
Comment (Fatmasiam) [inline README.md]: a
```suggestion
Website prototype is [here](https://www.figma.com/proto/Pom80HoRieQ24tSmTzayO4WP/City-of-south?node-id=291%3A782&scaling=min-zoom).
```

File: README.md [hunk 2]
```diff
@@ -50,10 +31,27 @@ South Londoners(Member):
...
+
+## How to use:
+
+- he project is still in builing stage..... In general:
```
Comment (Angham116) [inline README.md]:
```suggestion
- This project is still in the builing stage..... In general:
```

File: README.md (hunk 1 above)
Comment (AymanAlqoqa) [reply README.md]: I think it is ok.... no need to change url
thanks

File: README.md (hunk 2 above)
Comment (AymanAlqoqa) [reply README.md]: The refers to the project.
I'll correct typo
thanks you

**Pull request merged**